Testskript.R

```diff
@@ -8,4 +8,13 @@
 # Hallo Jule & Nicklas, das ist unser Testskript :-) Fuer mehr Smileys hier einmal ein: "JA"
 # :P
 
-# I want a dinosaur please! Preferably a tyrannosaurus
\ No newline at end of file
+# I want a dinosaur please! Preferably a tyrannosaurus
+
+Jule <- c(1, 2, 3, 4)
+Anne <- c(5, 6, 7, 8)
+
+Nicklas <- function(Jule, Anne) {
+  Nicklas <- (Jule + Anne)
+  return(Nicklas)
+}
+Nicklas(Jule, Anne)
```

Comment:
Wow, that looks very professional! I'm impressed!
Please change the number "8" to "9" in line 14. Thank you in advance!

Reply:
You can also put the comment directly below line 14 by clicking on the blue plus at line 14 :)

Reply:
ah nice, good to know!

Reply:
Thanks for the feedback, already adjusted the script!